pkg/scheduler/plugins/topology/job_filtering.go

```diff
@@ -333,18 +346,40 @@ func (*topologyPlugin) treeAllocatableCleanup(topologyTree *Info) {
 	}
 }
 
-func sortDomainInfos(domainInfos []*DomainInfo) []*DomainInfo {
-	sort.SliceStable(domainInfos, func(i, j int) bool {
-		if domainInfos[i].Level != domainInfos[j].Level {
-			return false
-		}
+func sortTree(root *DomainInfo, maxDepthLevel DomainLevel) {
```

Comment:
I recommend adding the function's purpose to the name. Something like sortBYDomainScore